src/Collection.ts

```diff
@@ -48,42 +48,34 @@ export default class Collection extends Base {
 
   /**
    * Specifies the model class for that collection
+   *
+   * @abstract
    */
-  model (attributes: { [key: string]: any } = {}): Class<*> {
-    return Model
-  }
+  abstract model (attributes?: { [key: string]: any }): new(attributes?: {[key: string]: any}) => T;
```

Comment:
👀 was is das?

Reply:
Model is an abstract method returning generic T that should be defined in child extending this parent class.

Reply:
omg, javascript syntax is getting out of hand haha